tests/common/strings_test.cpp

```diff
@@ -26,6 +26,22 @@
 
 using namespace Puzzles;
 
+TEST(Strings, PadLeading) {
+  EXPECT_EQ(padLeading("", 8, '0'), "00000000");
+  EXPECT_EQ(padLeading("1234", 8, '0'), "00001234");
+  EXPECT_EQ(padLeading("12345678", 8, '0'), "12345678");
+  EXPECT_EQ(padLeading("12345678", 8, '0'), "12345678");
```

Comment:
These are duplicates, aren't they?
```suggestion
  EXPECT_EQ(padLeading("12345678", 8, '0'), "12345678");
```